apps/reaper/test/integration/reaper/migrations_test.exs

```diff
@@ -12,7 +12,7 @@ defmodule Reaper.MigrationsTest do
     test "should pre-pend the brook instance to all scheduled quantum jobs" do
       Application.ensure_all_started(:redix)
 
-      {:ok, redix} = Redix.start_link(host: Application.get_env(:redix, :host), name: :redix)
+      {:ok, redix} = Redix.start_link(Keyword.put(Application.get_env(:redix, :args, []), :name, :redix))
```

Comment:
Nesting function calls like this can make Elixir a little unreadable. Consider using pipes like so:
```
{:ok, redix} =
    :redix
    |> Application.get_env(:args, [])
    |> Keyword.put(:name, :redix)
    |> Redix.start_link()
```